amr-wind/utilities/IOManager.H

```diff
@@ -132,6 +132,12 @@ private:
 
     //! Flag indicating whether we should allow missing restart fields
     bool m_allow_missing_restart_fields{true};
+
+#ifdef AMR_WIND_USE_HDF5
+    //! Flag indicating whether or not to output HDF5 plot files when HDF5 is
```

Comment:
can we modify this comment to fit on one line? Not sure how doxygen parses this...